docs/accounts.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Account Management
+sidebar_title: Accounts
+description: How to manage accounts addresses
+---
+
+Accounts are generated on Flow and not derived from private key. This creates an issue for testing, since 
+you need to create actors in a specific order to use their addresses properly.
+
+In order to reduce this friction we made a handy method `getAccountAddress`, which allows you to access specific address 
+by it is alias. This way you can think about actual actors - let's say usual crypto "suspects" `Alice` and `Bob` -  and 
+not which address they have.
+
+It also helps you to write tests in sequential and non-sequential way, since calling the method with the same 
+alias again will return existing address without creating new account.
```

Comment:
```suggestion
Flow accounts are not derived from a private key. This creates an issues for testing, since 
you need to create actors in a specific order to use their addresses properly.

In order to reduce this friction we made a handy method `getAccountAddress`, which allows you to access specific address using an alias. This way you can think about actual actors - for example `Alice` and `Bob` -  without needing to know their Flow addresses.

It also helps you to write tests in a sequential or non-sequential way. Calling this method for the first time will create a new account and return the address. Calling it a second time with the same alias again will return the Flow address for that account, without creating new account. 
```